modules/appeals_api/app/swagger/appeals_api/v1/notice_of_disagreements_controller_swagger.rb

```diff
@@ -641,6 +641,7 @@ class AppealsApi::V1::NoticeOfDisagreementsControllerSwagger
   swagger_path '/path' do
     operation :put, tags: NOD_TAG do
       key :operationId, 'putNoticeOfDisagreementEvidenceSubmission'
+      key :description, File.read(AppealsApi::Engine.root.join('app', 'swagger', 'appeals_api', 'v1', 'put_description.md'))
```

Comment:
I think with this inclusion we could remove the "example metadata.json file" section in our main doc body, or at least change it to point to this example at `/path`. What do you think?

Reply:
I think Kelly would like it to be in both spots... We can always remove it later if we decide too.